test/test_torch.py

```diff
@@ -17267,18 +17267,18 @@ def inner(self, device, dtype):
         1e-5, 1e-5, 1e-5, _float_types_no_half, _cpu_types, False),
     ('addcdiv', '', _small_2d,
         lambda t, d: [_small_2d(t, d),
-                      _small_2d(t, d, has_zeros=False)], 1, 1e-5, 1e-3,
-        _types, _cpu_types, True,
```

Comment:
@nairbv could you please review the precision changes?

Reply:
prec changes look fine.